run

```diff
@@ -7,6 +7,6 @@ env = ENV['ENVIRONMENT'] || 'development'.freeze
 
 Bundler.setup(:default, env.to_sym)
 
-require_relative 'github_graphql.rb'
+require_relative 'github_graphql_app.rb'
 
-GithubGraphql.start
+GithubGraphqlApp.new.start
```

Comment:
:( I always like it to just look pretty without that new, but I think this will be easier to change to a module and do